packages/loader/container-loader/src/container.ts

```diff
@@ -610,13 +609,6 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             assert(!!this.cachedAttachSummary,
                 "Summary should be there either by this attach call or previous attach call!!");
 
-            if (request.headers?.[DriverHeader.createNew] === undefined) {
```

Comment:
the point here is to have a common signal to url resolvers that they are resolving for the create new flow, not a load flow

Reply:
if you remove this, then i don't think there is a point to having any create new header, as url resolvers can't depend on its existence. in that case i wouldn't bother keeping a shared header name at all or augmenting the interface. personally, i think this removes a useful disambiguation for the url resolver to easily identify create new requests, and i don't see what problem it causes, but if you really want it gone, then we should just remove it entirely